src/Dofs/ConstraintHandler.jl

```diff
@@ -468,7 +468,7 @@ end
 
 # Saves the dirichlet boundary conditions to a vtkfile.
 # Values will have a 1 where bcs are active and 0 otherwise
-function WriteVTK.vtk_point_data(vtkfile, ch::ConstraintHandler)
+function WriteVTK.vtk_point_data(vtkfile, ch::ConstraintHandler{<: AbstractDofHandler,<: AbstractFloat})
```

Comment:
AFAIK VTK can just handle real-valued data natively. So other data types will require more special exports. Users will likely hit some kind of error way before coming this far, if they work with something different than float. However, I have not propagated <:AbstractFloat everywhere to see how far they come with their specific problem. I am not even sure what exactly to put into the docstrings to communicate this "restriction" properly.

Reply:
In this case, I think it would be better to put this somewhere else for writing to VTK. I won't include it in #692, but can be done as a follow-up on that and this PR to give better error messages in case they are bad. 

In this particular case, I don't think having the dofs as complex-valued is a problem because it should only write a zero or one, depending on whether the dof is constrained or not.

Reply:
Resolved by https://github.com/Ferrite-FEM/Ferrite.jl/pull/802/commits/fbdfa2339b9aa0857e4963d7bcf21aa753ded8fc